package.json

```diff
@@ -84,44 +86,33 @@
     "jest": "^27.0.1",
     "jest-config": "^27.0.1",
     "jest-puppeteer": "^5.0.4",
-    "lint-staged": "^11.0.0",
     "multihashing-async": "^2.1.2",
-    "prettier": "^2.3.0",
-    "prompt": "^1.1.0",
     "puppeteer": "^9.1.1",
-    "replace-in-file": "^6.2.0",
     "rimraf": "^3.0.2",
-    "rollup": "^2.50.3",
-    "rollup-plugin-gzip": "^2.5.0",
-    "rollup-plugin-node-polyfills": "^0.2.1",
-    "rollup-plugin-sourcemaps": "^0.6.3",
-    "rollup-plugin-terser": "^7.0.2",
-    "rollup-plugin-typescript2": "^0.30.0",
     "tslib": "^2.2.0",
-    "typedoc": "^0.20.36",
+    "typedoc": "^0.21.0-beta.2",
```

Comment:
Any significant changes in the beta?

Reply:
Yes. Typescript 4.3 support. I forgot why we are using typescript 4.3 now, though :sweat_smile:

Reply:
Oh and unrelated: There's a "library mode" now. We might be able to make use of that instead of running typedoc twice

Reply:
I thought we were already using that mode, but I'll reinvestigate to make sure 👀